openstack/identity/v3/tokens/requests.go

```diff
@@ -182,7 +182,7 @@ func Get(c *gophercloud.ServiceClient, token string) (r GetResult) {
 func Validate(c *gophercloud.ServiceClient, token string) (bool, error) {
 	resp, err := c.Request("HEAD", tokenURL(c), &gophercloud.RequestOpts{
 		MoreHeaders: subjectTokenHeaders(c, token),
-		OkCodes:     []int{200, 204, 400, 401, 403, 404},
+		OkCodes:     []int{200, 204, 400, 403, 404},
```

Comment:
The PR that implemented this is #203. This is was my misunderstanding of `OkCodes`. I was under the impression that any valid response (no matter success or failure), should be listed as a valid code.